pkg/apis/apps/types.go

```diff
@@ -180,6 +180,9 @@ type StatefulSetSpec struct {
 	// of the StatefulSet. Each pod will be named with the format
 	// <statefulsetname>-<podindex>. For example, a pod in a StatefulSet named
 	// "web" with index number "3" would be named "web-3".
+	// When the RestartPolicy field is specified in the Pod Template for StatefulSetSpec,
+	// only a .spec.template.spec.restartPolicy equal to Always is allowed,
+	// which is the default if not specified.
```

Comment:
perhaps abbreviate to `The only allowed template.spec.restartPolicy value is "Always".`

Reply:
Updated. Thanks.